.enb/techs/yml-source-reader.js

```diff
@@ -30,9 +31,17 @@ module.exports = require('./base-for-techs-with-modules')
          */
         _readYmlFileAndReplacePlaceholders: function (path, parameters) {
             return VowFs.read(path, 'utf8').then(function (content) {
+                var parametersInUse = {},
+                    processingContent;
+
                 for (var key in parameters) {
                     if (parameters.hasOwnProperty(key)) {
-                        content = content.replace(new RegExp('%' + key + '%', 'g'), parameters[key]);
+                        // we need this replace because % is reserved symbol in yml
+                        processingContent = content.replace(new RegExp('%' + key + '%', 'g'), 's%' + key + '%s');
```

Comment:
It would be better to combine all the keys to single RegExp and do replace only once